tests/unit/core/utils/test_compatibility.js

```diff
@@ -1115,12 +1127,6 @@ describe(__filename, () => {
   });
 
   describe('isFirefoxForIOS', () => {
-    it('returns false for Firefox Fenix', () => {
```

Comment:
😕 FirefoxForIOS was testing for Fenix but now isn't?

Reply:
As discussed below, any tests that existed for Fenix have now been combined into tests for Firefox for Android.